manifests/l/LesFerch/WinSetView/2.80/LesFerch.WinSetView.installer.yaml

```diff
@@ -0,0 +1,18 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: LesFerch.WinSetView
+PackageVersion: '2.80'
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: WinSetView-2.80/WinSetView.exe
+  PortableCommandAlias: WinSetView
+- RelativeFilePath: WinSetView-2.80/WinSetView.ps1
+  PortableCommandAlias: WinSetView.ps1

```

Comment:
@denelon should review this for compliance with MSFT policy

This also won't work currently, since the symlink will just open the script in an editor